diagnostic_remote_logging/package.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>diagnostic_remote_logging</name>
+  <version>0.0.1</version>
+  <description>diagnostic_remote_logging</description>
+  <maintainer email="dwijffels@lely.com">Daan Wijffels</maintainer>
+  <license>>BSD-3-Clause</license>
+
+  <buildtool_depend>ament_cmake</buildtool_depend>
+
+  <build_depend>diagnostic_msgs</build_depend>
+  <build_depend>rclcpp_components</build_depend>
+  <build_depend>libcurl4-openssl-dev</build_depend>
```

Comment:
```suggestion
  <build_depend>curl</build_depend>
```
According to https://github.com/ros/rosdistro/blob/c8060c60d888de2987192c50cdc077b94eb8a7a5/rosdep/base.yaml#L753